src/concept/type/RelationType.jl

```diff
@@ -7,5 +7,41 @@ end
 
 function create(x::RemoteConcept{RelationType})
     result = execute(x.transaction, RelationTypeRequestBuilder.create_req(x.concept.label))
-    return Relation(result.relation_type_create_res.relation)
+    return Relation(result.type_res.relation_type_create_res.relation)
+end
+
+function set_relates(transaction::AbstractCoreTransaction,
+                    label::Label,
+                    role_label::String,
+                    overridden_label::Optional{String} = nothing)
```

Comment:
How do we handle `nothing`?

Reply:
With the new PR#77 in ProtoBuf it is possible to send features containing nothing which doesn't crash the whole system.

Reply:
Ah you're right :-)